templates/Gemfile.erb

```diff
@@ -30,7 +30,7 @@ end
 
 group :development, :test do
   gem "bundler-audit", require: false
-  gem "byebug"
+  gem "pry-byebug"
```

Comment:
@patbenatar what happened to that alphabetical sort step you were going to add to the end? only partially kidding, since any build step can inject a new line in the Gemfile, keeping the starting template in alphabetical order is only a partial solution.